app/views/users/_form.html.erb

```diff
@@ -49,6 +49,13 @@
                     <% end %>
                 </div>
             </div>
+            <h3 class="p3"><%= t('users.daily_summary') %></h3>
+            <div class="field">
+                <div class="ui checkbox">
+                    <%= f.check_box :daily_summary %>
+                    <%= f.label :daily_summary, t('.daily_summary_intention') %>
```

Comment:
Aren't labels translated automatically? Why `t('.daily_summary_intention')` is needed here and why `intention` word is needed here?

Reply:
`daily_summary_intention` provides reason / intention for the user why this checkbox should be checked. Just like ToS.

Reply:
But why not just `daily_summary`? What happens if you remove `t` call completely? They should be translated automatically, as far as I know.

Reply:
Sure, it would auto-translate to "Daily summary to email" and "Igapäevane oksjoninimekiri meilile", but I had UX in mind with this one. Seemed nice touch to provide some additional, well-explained comment about this checkbox, but guess it's enough now. Removing it.